css-ui-3/caret-color-016.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Basic User Interface Test: caret-color visited link computed value</title>
+<link rel="author" title="Manuel Rego Casasnovas" href="mailto:rego@igalia.com">
+<link rel="help" href="http://www.w3.org/TR/css3-ui/#caret-color">
+<link rel="help" href="https://www.w3.org/TR/css3-color/#color0">
+<meta name="flags" content="dom interact">
```

Comment:
I am not entirely sure how to deal with a test case that needs manual set up (to visit the link) and has automated results via testharness.

@gsnedders do we have guidlines about this? Is the approach proposed in this test the right one?